gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,5 +2,6 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
 networkTimeout=10000
+validateDistributionUrl=true
```

Comment:
Why was this added? Were there problems?

Reply:
no this is a result of running the gradle wrapper
(shouldn't this be the case when running it?)

Reply:
I don't know. I guess this is perfectly fine. I just never observed something like this before.